server/src/KleeRunner.cpp

```diff
@@ -258,6 +260,8 @@ void KleeRunner::processBatchWithInteractive(const std::vector<tests::TestMethod
                                           "--bcov-check-interval=6s",
                                           "-istats-write-interval=5s",
                                           "--disable-verify",
+                                          "--check-div-zero=false",
+                                          "--check-overshift=false",
```

Comment:
We compile klee without any sanitizer, so don't turn off these options.

Reply:
Do you mean sanitizers for building KLEE? Because I mean sanitizers for building tested source code on which KLEE runs, take a look [here](https://github.com/UnitTestBot/UTBotCpp/blob/3cdb310e1802067d7705c2524f9163880b7850bb/server/src/utils/SanitizerUtils.cpp#L12)

Reply:
Do we turn on ub sanitizer for launch klee on tested code? I can't find this moment in the code.

Reply:
Well, let me explain how it works and why KLEE doesn't need to be compiled with `-fsantize=*` options itself.

- UTBot compiles "_klee" files with `-fsanitize=*` options [here](https://github.com/UnitTestBot/UTBotCpp/blob/main/server/src/KleeGenerator.cpp#L139)
- KLEE accepts "sanitized" code and adds special handlers [here](https://github.com/UnitTestBot/klee/blob/utbot-main/lib/Core/SpecialFunctionHandler.cpp#L145)
- When the division by zero occurs in the given code, one of the handlers triggers and generate "error" test [here](https://github.com/UnitTestBot/klee/blob/utbot-main/lib/Core/SpecialFunctionHandler.cpp#L1059)

Speaking about the '--check-div-zero' option, additional instrumentation on the part of KLEE is unnecessary. Moreover, it takes time in order to instrument LLVM IR code and then generates unreachable branches [here](https://github.com/UnitTestBot/klee/blob/utbot-main/lib/Module/KModule.cpp#L322)